src/mctpd.c

```diff
@@ -2895,6 +2895,48 @@ static int bus_link_get_prop(sd_bus *bus,
 	return rc;
 }
 
+static int notify_discovery(ctx *ctx, int ifindex)
+{
+	int rc = 0;
+	dest_phys desti = { 0 }, *dest = &desti;
+	struct mctp_ctrl_cmd_discovery_notify req = { 0 };
+	struct mctp_ctrl_resp_discovery_notify *resp;
+	uint8_t *buf;
+	size_t buf_size;
+	struct sockaddr_mctp_ext resp_addr;
+
+	dest->ifindex = ifindex;
+	mctp_nl_ifaddr_byindex(ctx->nl, dest->ifindex, &dest->hwaddr_len);
+	memset(dest->hwaddr, 0, sizeof dest->hwaddr);
```

Comment:
A zero hwaddr is specific to PCIe, right? Do we also need to know the physical broadcast address for this link type?